### DISCUSSION TITLE
Have "for" in the current-state node work for <=, >=, ...

### DISCUSSION THREAD

**Discussion by Hadatko** [zachowj/node-red-contrib-home-assistant-websocket#1216] [Feature Request]
### Describe the bug

It looks like for half of IF conditions for current state node cannot be set FOR value for following value in timeout

### To Reproduce

Edit any current state node. Try is, is not, in, not in  and compare with <=, >=, ...

### Expected behavior

FOR option should be available for all options or none of them.

### Screenshots

![image_2023-12-23_13-22-42](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/13130700/0b3d541e-93da-48dd-b261-4adb2be19f45)
![image_2023-12-23_13-23-14](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/13130700/7cbb704b-9181-4db7-ac50-96d98e835edb)

### Environment Information

Version:  latest one

Home Assistant version: latest one
Companion version:

Node-RED version: latest one
Docker: yes
Add-on: no

Node.js version:
OS:

**Comment by zachowj** [comment]
Because it gets how long it's been in a given state based on the last changed timestamp. With <, >, ... the state value could be `true` on multiple states but the last timestamp would keep changing and the beginning time is unknown.

Could this be changed to work with <, >, yes.

**Comment by Hadatko** [comment]
Oh i see. But you closed it to early :D Are you willing to implement support for this?

**Comment by Hadatko** [comment]
For example node event-state supports all cases.

**Comment by zachowj** [comment]
Sure it could be implemented. It currently isn't on my list but it would be a welcomed addition.

Moving this to feature requests.

**Comment by Hadatko** [comment]
Thank you very much. Also for such quick  response.

**Comment by ciotorcristian** [comment]
Any updates on this? It would be a welcomed feature to have this option.
